lib/components/primitive-components/Group/Group.ts

```diff
@@ -218,6 +207,8 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
           this._asyncAutoroutingResult = {
             output_pcb_traces: autorouting_job_output.output_pcb_traces,
           }
+          // Apply the routing results immediately before the next autorouting job starts
+          this._updatePcbTraceRenderFromPcbTraces()
```

Comment:
we're not suppose to touch circuit json while inside of an effect (this is a rule of react, but unfortunately we don't enforce it currently)